components/server/src/code-sync/code-sync-service.ts

```diff
@@ -365,58 +296,34 @@ export class CodeSyncService {
         }
 
         let content: string;
-        if (resourceRev === fromTheiaRev) {
```

Comment:
there are still some references to fromTheiaRev above, but not sure about semantic, there are should be unit tests though